assignments/prototypes.js

```diff
@@ -102,7 +129,7 @@
   console.log(archer.greet()); // Lilith offers a greeting in Elvish.
   console.log(mage.takeDamage()); // Bruce took damage.
   console.log(swordsman.destroy()); // Sir Mustachio was removed from the game.
-*/
+
 
```

Comment:
Make sure you seek disambiguation on the Object.create and .call (as a constructor) methods/patterns.

Did todays lecture help you though?